Week3

ashwanthdurairaj opened the pull request:
Create week 3 assignment for review

ashwanthdurairaj committed:
create week3 assignment

ashwanthdurairaj committed:
update app.js

ashwanthdurairaj committed:
modify app.js

DonBrowny commented on week3/src/App.js:
```diff
@@ -0,0 +1,130 @@
+import {useState} from 'react'
+
+function Square({value, onSquareClick})
```
create a file names `square.jsx` and move the Squarer component code to it.

DonBrowny commented on week3/src/App.js:
```diff
@@ -0,0 +1,130 @@
+  )
+}
+
+function Board({xIsNext, squares, onPlay})
```
create a file names `board.jsx` and move the Squarer component code to it.

DonBrowny commented on week3/src/App.js:
```diff
@@ -0,0 +1,130 @@
+  )
+}
+
+function calculateWinner(squares) {
```
create a file names `utile/calculate-winner.js` and move the calcaulteWinner function to the new file.

DonBrowny reviewed (changes requested):
Convert this project from npm to yarn. Also try adding [prettier](https://yarnpkg.com/package?name=prettier) package to this repo and modify your VS code to use  prettier

ashwanthdurairaj committed:
convert project from npm to yarn, add prettier and create components and utils directory

ashwanthdurairaj committed:
update calculate-winner

ashwanthdurairaj commented:
converted project from npm to yarn, added prettier to the existing project, moved all components into a separate component folder and functions into a separate utils folder

ashwanthdurairaj committed:
update calculate-winner

ashwanthdurairaj committed:
update board

ashwanthdurairaj committed:
update calculate-winner
